src/NosCore.GameObject/Networking/ClientSession/ClientSession.cs

```diff
@@ -161,8 +161,8 @@ public override void ChannelRead(IChannelHandlerContext context, object message)
             {
                 return;
             }
-
-            HandlePackets(buff, context);
+            //https://github.com/Azure/DotNetty/issues/265
+            Task.Run(async () => { await HandlePackets(buff, context); });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)